v2/client_test.go

```diff
@@ -1,21 +1,21 @@
-package tailscale_test
+package tailscale
```

Comment:
I had to change this to gain access to the unexported `buildTailnetURL` function.